recipes-bsp/xt-rpi-u-boot-scr/xt-rpi-u-boot-scr.bb

```diff
@@ -11,14 +11,18 @@ TEMPLATE_FILE = "boot.cmd.xen.in"
 
 SRC_URI = "file://${TEMPLATE_FILE}"
 
+DOMD_MMC_DT_NAME = "${RPI_SOC_FAMILY}-${MACHINE}-mmc.dtb"
+DOMD_USB_DT_NAME = "${RPI_SOC_FAMILY}-${MACHINE}-usb.dtb"
+DOMD_BOOT_DEVICE = "${@bb.utils.contains('MACHINE_FEATURES', 'domdusb', '\\/dev\\/sda2', '\\/dev\\/mmcblk0p2', d)}"
```

Comment:
I believe it is better to do via device tree overlays.

Reply:
Right. Possible Idea is to have mostly empty bcm2712-raspberrypi5-domd.dts (root, `gic` and `passthrough` with cells/ranges/compatible) +
```
 bcm2712-raspberrypi5-domd-mmc.dtso,
 bcm2712-raspberrypi5-domd-usb.dtso,
 bcm2712-raspberrypi5-domd-net.dtso
```
u-boot reads - bcm2712-raspberrypi5-domd.dts and applies -mmc|usb.dtso + -net.dtso depending on MACHINE_FEATURES.

There could be common parts of-course 

Super ideal result is to be able to do smth. like this:
`moulin X --ENABLE_DOMD_USB ----ENABLE_DOMD_NET`

Reply:
> Super ideal result is to be able to do smth. like this:
This already implemented. I have appropriate PR